app/src/main/java/com/monkeyteam/chimpagne/viewmodels/EventViewModel.kt

```diff
@@ -279,11 +365,12 @@ data class EventUIState(
     val supplies: Map<ChimpagneSupplyId, ChimpagneSupply> = mapOf(),
     val parkingSpaces: Int = 0,
     val beds: Int = 0,
-
     // unmodifiable by the UI
     val ownerId: ChimpagneAccountUID = "",
     val currentUserRole: ChimpagneRole = ChimpagneRole.NOT_IN_EVENT,
-    val loading: Boolean = false,
+    // OPTIONAL PARAMETER, MUST BE FETCH SEPARATELY
+    val accounts: Map<String, ChimpagneAccount?> = emptyMap(),

```

Comment:
this is part of AccountViewModel, remove it

Reply:
Yes indeed, thank you for pointing it out. This will also remove some burden from the already bloated EventViewModel file.